test/controllers/sessions_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
```

Comment:
@jynti Please remove all test files. And do not use rails generators for models and controllers.